tensorflow/lite/micro/examples/hello_world/quantization/ptq.py

```diff
@@ -61,73 +58,30 @@ def get_data():
   # Shuffle the values to guarantee they're not in order
   np.random.shuffle(x_values)
 
-  # Calculate the corresponding sine values
-  y_values = np.sin(x_values).astype(np.float32)
-
-  return (x_values, y_values)
-
-
-def create_model() -> tf.keras.Model:
-  model = tf.keras.Sequential()
-
-  # First layer takes a scalar input and feeds it through 16 "neurons". The
-  # neurons decide whether to activate based on the 'relu' activation function.
-  model.add(tf.keras.layers.Dense(16, activation='relu', input_shape=(1, )))
-
-  # The new second and third layer will help the network learn more complex
-  # representations
-  model.add(tf.keras.layers.Dense(16, activation='relu'))
-
-  # Final layer is a single neuron, since we want to output a single value
-  model.add(tf.keras.layers.Dense(1))
-
-  # Compile the model using the standard 'adam' optimizer and the mean squared
-  # error or 'mse' loss function for regression.
-  model.compile(optimizer='adam', loss='mse', metrics=['mae'])
+  return x_values
 
-  return model
 
-
-def save_tflite_model(tflite_model, save_dir, model_name):
+def save_tflite_model(tflite_model, target_dir, model_name):
```

Comment:
let's use flatbuffer_utils.write_model instead of a new function in this file. The same comment might hold for train.py as well

https://github.com/tensorflow/tflite-micro/blob/b4fe6691bf95d933b4a96f36bb1a03a979ebc257/tensorflow/lite/tools/flatbuffer_utils.py#L92-L104